src/popup/components/TransferSendFormBase.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="transfer-send-base">
+    <template v-if="$slots.header">
+      <slot
+        name="header"
+      />
+    </template>
+    <template v-else>
+      <ModalHeader :title="customTitle || $t('modals.send.sendTitle')" />
+      <div class="account-row">
+        <AccountItem
+          :address="activeAccount.address"
+          :protocol="activeAccount.protocol"
+          :name="activeAccount.name"
+          size="md"
+        />
+      </div>
+    </template>
+
+    <slot name="recipient" />
+
+    <slot name="amount" />
+
+    <slot name="extra" />
+
+    <DetailsItem :label="$t('transaction.fee')">
+      <template #value>
+        <TokenAmount
+          :amount="fee"
+          :symbol="feeSymbol"
+          :protocol="protocol"
+          data-cy="review-fee"
+        />
+      </template>
+    </DetailsItem>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent,
+  PropType,
+} from 'vue';
+import { useStore } from 'vuex';
+import { useAccounts } from '@/composables';
+import type { Protocol } from '@/types';
```

Comment:
Please try always to put the types import as the first import of the repo files (after external packages).